package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-aria-modal",
-  "version": "2.11.1",
+  "version": "2.11.2",
```

Comment:
Let's remove this change from the PR. I prefer to set the version number when I'm ready to cut a release, rather than as a part of PRs.